src/actions.ts

```diff
@@ -333,6 +334,27 @@ export function getActions(instance: PtzOpticsInstance): CompanionActionDefiniti
 				void instance.sendCommand(PresetRecall, event.options)
 			},
 		},
+		[PtzOpticsActionId.RecallPresetFromVar]: {
+			name: 'Recall Preset (by number)',
+			options: [
+				{
+					type: 'textinput',
+					label: 'Preset Number',
+					id: PresetRecallOption.id,
+					useVariables: true,
+					tooltip: 'Preset number range of 0-89, 100-254',
+					default: '0',
+				},
+			],
+			callback: async ({ options }, context) => {
+				const errorOrValue = await parsePresetVariableOption(options, context)
```

Comment:
Name this `presetRecallOptions`.

Reply:
Changed the ts file name to be generic since the same will apply equally to the save preset from number and left this parse function as is since will apply equally to both